website/content/docs/connect/config-entries/service-defaults.mdx

```diff
@@ -625,6 +669,31 @@ spec:
         },
       ],
     },
+    {
+      name: 'Destination',
+      type: 'DestinationConfig: <optional>',
+      description: `Controls configuration specific to destinations through terminating-gateway. Added in v1.13.0.`,
+      children: [
+        {
+          name: 'Addresses',
+          type: 'array<string>: []',
+          description: {
+            hcl:  `List of addresses associated with the destination. This can be a hostname or an IP address.
+          Wildcards are not accepted.`,
+            yaml:  `List of addresses associated with the destination. This can be a hostname or an IP address.
+          Wildcards are not accepted.`,
+          },
```

Comment:
If they're the same description you can just use `description: "blah"`. You don't need the subkeys, those are only for when the descriptions are different.

Reply:
if yaml is false that means there is no kube description so then yeah you can just use description, if yaml is true then if they're the same description don't use the subkeys and if they're different use the subkeys

Reply:
and I guess yaml is true by default, right?